plugins/home/src/plugin.ts

```diff
@@ -210,3 +211,18 @@ export const HomePageRecentlyVisited = homePlugin.provide(
       import('./homePageComponents/VisitedByType/RecentlyVisited'),
   }),
 );
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = homePlugin.provide(
+  createReactExtension({
```

Comment:
```suggestion
  createComponentExtension({
```

createReactExtension is a low level helper for any react related concerns; actual components use this one

Reply:
Ok, I originally had `createComponentExtension` and was suggested to use `createReactExtension` [here](https://github.com/backstage/backstage/pull/20350#discussion_r1347172369)
I'll switch it back

Reply:
Hey @nikolarAutodesk I think I'm essentially not entirely sure about the original motivation for the comments that led to this. @drodil said *IF* you want to, etc. But is that really the case? Do you really strongly want a ReactNode title on this or can we just side step that entire problem and keep things as vanilla as possible and use `createCardExtension` which I expect is the originally intended one to leverage? Otherwise it's "just another component" and can't be placed on actual composable home pages.

So I'd like to vote to use `createCardExtension` and sidestep the type issue, then later on work toward ReactNode titles if that's really a feature that's particularly desired. I have my doubts that it's generally a good idea to put things that freely in specifically titles, to be honest.

Reply:
What's the reason to not use `createCardExtension`? It's most likely what we want to be able to use here and if there's something preventing that we may want to change `createCardExtension` instead. Otherwise if this is a plain component you should use `createComponentExtension`. Do not use `createReactExtension` as it's a lower level API that we don't want to be used directly, it's for example what's used to implement `createCardExtension` under the hood.

Reply:
@freben @Rugvip thanks for the responses, I can switch to `createCardExtension`, though
our team does strongly leverage the ability to pass in a `ReactNode` title, and if `createCardExtension` uses `InfoCard` ([here](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/plugins/home-react/src/extensions.tsx#L171)) under the hood, then shouldn't it also allow title to be a `ReactNode` rather than a `string`([here](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/plugins/home-react/src/extensions.tsx#L82))?

This is my original response for avoiding `createCardExtension`:

> I went with `createReactExtension` specifically because of types.
> I think there are a few changes that need to be made in `plugins/home-react/src/extensions.tsx` and some other files to avoid ts errors when allowing a `ReactNode` to be passed for the title as intended in [InfoCard](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/packages/core-components/src/layout/InfoCard/InfoCard.tsx#L133).
> We noticed this when we decided to use [StarredEntities](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/plugins/home/src/plugin.ts#L169) in our own Home Page. We plan on contributing this fix in the future if it is not already under way 👍

Reply:
updated 👍